templates/vue3/src/main.js

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue'
+import App from './App.vue'
+
+Vue.config.productionTip = false
```

Comment:
What’s this? Should it be configured with an environment variable?

Reply:
It's standard to set it to false. Just stops an annoying "tip" from appearing every time you run. https://stackoverflow.com/questions/41743926/disable-development-mode-warning-in-vuejs